tests/framework/contracts/operations.go

```diff
@@ -88,5 +88,6 @@ type RestAPIOperator interface {
 }
 
 type TestOperator interface {
-	TearDown()
+	SetUpRook()
```

Comment:
Do you need `Rook` in the method names? Seems implicit.

Reply:
it is implicit. Removed `Rook`